cpp/FLIP.h

```diff
@@ -2183,7 +2184,7 @@ namespace FLIP
             this->setState(CudaTensorState::DEVICE_ONLY);
         }
 #endif
-        void computeExposures(std::string tm, float& startExposure, float& stopExposure)
+        void computeExposures(const std::string tm, float& startExposure, float& stopExposure)
```

Comment:
you can use & on the `const std::string tm` i.e., `const std::string& tm`

Reply:
Fixed!